cli/central-client/src/main/java/org/ballerinalang/central/client/CentralAPIClient.java

```diff
@@ -284,11 +288,11 @@ public Package getPackage(String orgNamePath, String packageNamePath, String ver
      */
     public List<String> getPackageVersions(String orgNamePath, String packageNamePath, String supportedPlatform,
             String ballerinaVersion) throws CentralClientException {
-        String packageSignature = orgNamePath + "/" + packageNamePath;
+        String packageSignature = orgNamePath + SEPARATOR + packageNamePath;
         Optional<ResponseBody> body = Optional.empty();
         OkHttpClient client = this.getClient();
         try {
-            String resourceUrl = "/" + PACKAGES + "/" + orgNamePath + "/" + packageNamePath;
+            String resourceUrl = SEPARATOR + PACKAGES + SEPARATOR + orgNamePath + SEPARATOR + packageNamePath;
```

Comment:
`resourceUrl` seems to be defined and used in multiple places throughout the class. So maybe we can have it as a class variable

Reply:
Fixed with 010f5dc07ab

Reply:
Although it is defined and used in each function, it holds different values (URLs for each endpoint) in each of them. It doesn't make much sense to have it defined at class level.